tripal/src/Services/TripalCitationManager.php

```diff
@@ -0,0 +1,103 @@
+<?php
+
+namespace Drupal\tripal\Services;
+
+
+/**
+ * The citation manager class is used to generate citations for
+ * publications of various types. The primary function is
+ * generateCitation(), and input consists of two parts, a
+ * citation template containing various tokens in a particular
+ * order with appropriate punctuation, and an associative array
+ * of key => value pairs that will be used to replace the
+ * tokens in the template.
+ *
+ * This class can also provide default templates for several
+ * types of publications.
+ *
+ */
```

Comment:
Docblock standards should have a single line summary first.
```suggestion
/**
 * Generates citations for publications.
 *
 * The citation manager class is used to generate citations for
 * publications of various types. The primary function is
 * generateCitation(), and input consists of two parts, a
 * citation template containing various tokens in a particular
 * order with appropriate punctuation, and an associative array
 * of key => value pairs that will be used to replace the
 * tokens in the template.
 *
 * This class can also provide default templates for several
 * types of publications. See getDefaultCitationTemplate() for
 * the types of publications supported.
 */
```